src/grammar/grammar.ohm

```diff
@@ -102,19 +102,25 @@ Tact {
     StatementBlock = "{" Statement* "}"
     StatementLet = let id ":" Type "=" Expression ";"
     StatementReturn = return Expression ";" --withExpression
-                    | return ";" --withoutExpression    
+                    | return ";" --withoutExpression
     StatementExpression = Expression ";"
     StatementAssign = LValue "=" Expression ";"
     StatementAugmentedAssign = StatementAugmentedAssignAdd
                              | StatementAugmentedAssignSub
                              | StatementAugmentedAssignMul
                              | StatementAugmentedAssignDiv
                              | StatementAugmentedAssignRem
+                             | StatementAugmentedAssignOr
+                             | StatementAugmentedAssignAnd
+                             | StatementAugmentedAssignXor
```

Comment:
These identifiers should have `Bitwise` in them for consistency with the rest of the grammar